backend/WebApplication1/Services/FaceService.cs

```diff
@@ -0,0 +1,405 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using WebApplication1.Models.FaceAPI;
+using WebApplication1.Models.FaceAPI.Responses;
+using WebApplication1.Models.FaceAPI.Requests;
+
+namespace WebApplication1.Services
+{
+    public class FaceService
+    {
+        private const string _subscriptionKey = "822fc0c2b7704dd48003c050650f4522";
+        private const string _ocpApimSubscriptionKey = "Ocp-Apim-Subscription-Key";
+        private const string _uriBase = "https://westeurope.api.cognitive.microsoft.com/face/v1.0";
+
+        public static async Task<List<PersonGroupGetResponse>> GetPersonGroups()
+        {
+            using (HttpClient client = new HttpClient())
+            {
+                client.DefaultRequestHeaders.Add(_ocpApimSubscriptionKey, _subscriptionKey);
+                string uri = $"{_uriBase}/persongroups";
+                HttpResponseMessage response = await client.GetAsync(uri);
+                if (response.IsSuccessStatusCode)
+                {
+                    string content = await response.Content.ReadAsStringAsync();
+                    List<PersonGroupGetResponse> result = JsonConvert.DeserializeObject<List<PersonGroupGetResponse>>(content);
+                    return result;
+                }
+                else
+                {
+                    string errorText = await response.Content.ReadAsStringAsync();
+                    FaceApiErrorResponse errorResponse = JsonConvert.DeserializeObject<FaceApiErrorResponse>(errorText);
+                    throw new FaceApiException(errorResponse.Error.Code, errorResponse.Error.Message);
+                }
+            }
+        }
+
+        public static async Task<PersonGroupGetResponse> GetPersonGroup(string personGroupId)
```

Comment:
move all the methods, that are dependent on Face API to a new service, FaceApiService or smth like that, because FaceService should only process the logic from repository data, all other functionality should be delegated to other classes